tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -611,6 +618,13 @@ def from_tensor_slices(tensors):
            [3, 2]], dtype=int32), array([[b'A'],
            [b'B']], dtype=object))
 
+    >>> # `from_tensor_slices` merges the input tensor 
+    >>> # unlike `from_tensors` which will create 3D tensor in below example.
```

Comment:
Merge these two lines in a single comment line

```
>>> # `from_tensor_slices` merges the input tensor, unlike `from_tensors` which increases dimensionality
```

(or two lines if it doesn't fit)